### DISCUSSION TITLE
Webmail SSL WRONG_VERSION_NUMBER

### DISCUSSION THREAD

**Discussion by nickshanks347** [modoboa/modoboa#2211] [Help]
Hi, so I thought I'd ask in support before making an issue but when I click on webmail under the modoboa interface, I get this error:
**`Error: Connection to IMAP server failed: [SSL: WRONG_VERSION_NUMBER] wrong version number (_ssl.c:1123)`**

I have just downloaded and ran the most recent install script but this hasn't made a difference, it still shows this error. 

Does anyone have any ideas?

**Comment by tonioo** [comment]
Can you check the webmail configuration? (Go to Modoboa > Parameters > Webmail).
